tests/python/PEP8NotebookBearTest.py

```diff
@@ -77,6 +77,35 @@
 }
 """
 
+# metadata field deleted manually
+inf_line_file = r"""{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "x = 'This is quite a long string and it should not ",
```

Comment:
Is this the correct way of splitting the source into multiple files? If it is, I guess there is some problem in the testing then.

Reply:
I tried it locally, it goes correctly. Not sure what wen't wrong with the checking the file.